R/flatten.R

```diff
@@ -1,58 +1,53 @@
-#' Flatten part of a `dm` into a wide table
+#' Flatten a part of a `dm` into a wide table
 #'
-#' Gather all information of interest in one place in a wide table (on a database-[`dm`] a
-#' temporary table will be created).
-#' If referential integrity is given among the tables of the data model, the resulting
-#' table of this function will contain as many rows as the table `start` does (exceptions are
-#' `join = anti_join` (result is empty table with same columns as `start`) and `join = right_join`
-#' (number of rows equal to or larger than those of `start`)).
+#' Gather all information of interest in one place in a wide table - a temporary table that will be created on a database-[`dm`].
+#' If referential integrity is specified for the tables of the data model, the table that results from the flattening function
+#' will contain as many rows as the `start` table does (exceptions are
+#' `join = anti_join` (the result is an empty table with the same columns as `start`) and `join = right_join`
+#' (the number of rows is equal to or larger than those of `start`)).
 #' For more information please refer to `vignette("dm-joining")`.
 #'
 #' @inheritParams cdm_join_to_tbl
-#' @param start Table to start from. From this table all outgoing foreign key relations are
-#' considered to establish a processing order for the joins. An interesting choice could be
-#' for example a fact table in a star schema.
-#' @param ... Unquoted table names to include in addition to `start`. The order of the tables here determines
-#' the order of the joins. If empty, all tables that can be reached are included.
-#' If this includes tables which aren't direct neighbours of `start`,
+#' @param start The table from which all outgoing foreign key relations are considered
+#' in order to establish a processing order for the joins.
+#' An interesting choice would be, e.g. a fact table in a star schema.
+#' @param ... Unquoted names of the tables to be included in addition to the `start` table. The order of the tables here determines
+#' the order of the joins. If the paramater is empty, all tables that can be reached will be included.
```

Comment:
The term "parameter" is overloaded in the stats community.

```suggestion
#' the order of the joins. If the argument is empty, all tables that can be reached will be included.
```